packages/matter.js/src/session/SecureSession.ts

```diff
@@ -217,16 +230,25 @@ export class SecureSession<T> implements Session<T> {
     }
 
     /** Ends a session. Outstanding subscription data will be flushed before the session is destroyed. */
-    async end(sendClose: boolean) {
+    async end(sendClose: boolean, delayClose = false) {
```

Comment:
"delayClose" is confusing, it makes it sound like an arbitrary timeout or something.  Maybe "deferClose" or "closeAfterExchange"?

Reply:
also renamed "closingAfterExchangeFinished"